portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,42 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+  const funfacts =
+      ['I love hiking, biking, and spending time outdoors.', 
+      'At Columbia, I am part of the Pops Orchestra, an ensemble dedicated to arranging and playing pop pieces for orchestra.',
+       'I have two pet cats.',
+        'My favorite movie is The Matrix.', 
+        'I am a middle child - I have an older brother and two younger sisters.', 
+        'My favorite food is garbanzo beans.', 
+        'Currently, I am attempting to grow potatoes outside.',
+        'My favorite animal is a panda.'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const funfact = funfacts[Math.floor(Math.random() * funfacts.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  // Add it to the page in a talbe.
+  var funFactTable = document.getElementById('funfact-table');
+  const funFactRowText = '<tr><td>' + funfact + '</td></tr>';
+
+  if(!tableText.includes(funFactRowText)){
+        tableText += '<tr><td>' + funfact + '</td></tr>';
+        countTableEntries++;
+        document.getElementById('funfact-message').innerHTML = "";
+  }else if(countTableEntries < funfacts.length){
+      document.getElementById('funfact-message').innerHTML = "Repeat fact - try again!";
+  }else{
+      //need to deal with edge case that the table has all the entries more elegantly
+      document.getElementById('funfact-message').innerHTML = "Congrats - you've gone through all of the fun facts!";
+  }
+  
```

Comment:
Maybe also remove blank line here. See https://google.github.io/styleguide/jsguide.html#formatting-whitespace

Reply:
Sorry I misread. No need to remove.